src/types.ts

```diff
@@ -84,6 +84,8 @@ export interface ChangelogOptions extends Partial<ChangelogenOptions> {
    * Release repository, defaults to `repo`
    */
   releaseRepo?: RepoConfig | string
+
+  tag?: string
```

Comment:
Could you add a jsdoc to explain how this works?

Reply:
https://github.com/croatialu/changelogithub/blob/fix/last-tag/src/types.ts#L88-#L96